integ-test/src/test/java/org/opensearch/sql/sql/RelevanceFunctionIT.java

```diff
@@ -35,4 +35,17 @@ void match_in_having() throws IOException {
     verifyDataRows(result, rows("Bates"));
   }
 
+  @Test
+  void match_phrase_in_where() throws IOException {
+    JSONObject result = executeQuery("SELECT firstname WHERE match_phrase(lastname, 'Bates')");
```

Comment:
Don't these SELECT statements need FROM clauses?

Reply:
You are right -- removed until the tests work.